build/avalanchego-apis/exchange-chain-x-chain-api.md

```diff
@@ -619,7 +621,24 @@ curl -X POST --data '{
     "method" :"avm.export",
     "params" :{
         "to":"C-avax1q9c6ltuxpsqz7ul8j0h0d0ha439qt70sr3x2m0",
-        "amount": 500,
+        "amount": 10,
```

Comment:
Did you mean to put 2 curl calls here?

Reply:
yeah but it seems weird, just removed it.